src/acceleration/test/AccelerationIntraCommTest.cpp

```diff
@@ -67,16 +67,16 @@ BOOST_AUTO_TEST_CASE(testVIQNILSpp)
   mesh::PtrData displacements(new mesh::Data("dvalues", -1, 1));
   mesh::PtrData forces(new mesh::Data("fvalues", -1, 1));
 
-  if (context.isPrimary()) { //Primary
+  if (context.isPrimary()) { // Primary
```

Comment:
These little changes are really a nightmare when a merge-conflict occurs. Looks to me like our linter has a blind spot here. Can we somehow fix this @fsimonis ?